### DISCUSSION TITLE
Duty Outfits make the Client crash

### DISCUSSION THREAD

**Discussion by melliiii** [OwlGamingCommunity/V#33] [Support]
Heiii Guys, im trying to get this to work, i already fixed some issues but i dont know where this comes from... Im not that good at C#, more of a Java programmer :D Maybe someone can help or ill post my findings here later if i can resolve it on my own:

When i create a Official Faction (via Database, type 0), for example a Sheriffs Department and create a duty point with /createduty 1, then go on duty and try to create a outfit or just select on i get this error and im stuck in the screen... Then i have to restart the server and everything is back to normal...

Also if i then select the just created outfit and go on duty i cant move, but i leave the screen. It gives the same error message

```
============== CLIENTSIDE EXCEPTION ==============
GOT CLIENTSIDE EXCEPTION FROM: melli
Deserialize Failed, identifier was HairAndTattoos, input was NULL

        OwlJSON.DeserializeObject[T](String strJSON, EJsonTrackableIdentifier identifier, String optionalIdentifierAddon) in cs_packages/owl_core.client/OwlJSON.Client.cs:line 132

        SkinHelpers.ApplyTattoosAndHairTattoosForPlayer(Player player, Boolean bForceHideHair, Boolean bUseForceBaseHairID, Int32 forceBaseHairID) in cs_packages/owl_core.client/SerializationDefinitions.Client.cs:line 892

        GenericCharacterCustomization.ApplyComponent(ECustomClothingComponent component) in cs_packages/owl_stores.client/GenericCharacterCustomization.Client.cs:line 418

        GenericCharacterCustomization.ApplyClothesToggleState() in cs_packages/owl_stores.client/GenericCharacterCustomization.Client.cs:line 452

        DutyOutfitEditor.OnEnterDutyOutfitEditorResponse(List`1 lstOutfits) in cs_packages/owl_factions.client/DutyOutfitEditor.Client.cs:line 124
==========================================
```

**Comment by PolishDywan** [comment]
BUMP :(

**Comment by Bullsito** [comment]
Hi.

Did you modify any HairTattooData.json or TattooData.json file?

It looks like to be an issue with the tattoos/hair tattoos JSON files when they tried to set it.

(I already talked with PolishDywan anyway, but feel free to comment here)
